templates/strapdown.html.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<body>
```

Comment:
Also, we should probably still have a `<head>` above the body with `<title>` inside so that when users open the page, there is something shown on the tab in their browser. try `{{title}}`?

Reply:
@Xenorith suggested removing the title because it making jumping by anchor tags difficult due to the title bar covering the sub section title.

Reply:
the header is completely different from `<head><title></title></head>`. This is page metadata only and does not affect rendering of the page. It will just put the page title in your tab of the browser

Reply:
The CSS will automatically add a header bar if there is a title present.

Reply:
This is the worst piece of javascript I've ever encountered